jupyterlab/staging/yarn.lock

```diff
@@ -6370,7 +6370,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"handlebars@npm:4.7.7, handlebars@npm:^4.5.3":
+"handlebars@npm:4.7.7":
   version: 4.7.7
```

Comment:
The old version is still in the lockfile.

Reply:
Ah thanks. Looks like verdaccio is messing with me.  I will update that dep, too.

Reply:
It turns out it's not as easy as i'd hoped. we could go down the rabbit hole of updating deps one by one, but there is also the option of doing a wholesale upgrade of all the deps. I will keep this pr focused on handlebars, but make another PR to upgrade all the deps.

Reply:
It's ok, let's just rename the PR to reflect that and de-duplicate the lockfile (e.g. `jlpm dlx yarn-berry-deduplicate`) to avoid having two versions